packages/app-bridge/src/registries/api/GetProxyNetworkCall.ts

```diff
@@ -0,0 +1,12 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type GetProxyNetworkCallPayload = {
+    id: string;
+    props: unknown;
+};
+
+export type GetProxyNetworkCallResponse = {
+    data: unknown;
+};
+
+export const getProxyNetworkCall = (payload: GetProxyNetworkCallPayload) => ({ name: 'getProxyNetworkCall', payload });
```

Comment:
Proxy network call would mean we have in place a [Proxy](https://wikipedia.org/wiki/Proxy) in place which makes the request to some hosts instead of the current user. Is that the case?

Reply:
Yes, To be more precise we will have one. (it's in development phase). We'll have a secret-proxy-service to handle secure API tokens and calls to third party. This PR is just one small piece of our Secure Settings feature.